### DISCUSSION TITLE
Is 'libfox' still used?

### DISCUSSION THREAD

**Discussion by philmoz** [EdgeTX/edgetx#2857] [Q&A] [answered]
I noticed that libfox is not added to the MacOS companion app bundle and the app happily runs on MacOS without installing libfox.

I removed libfox from my main Mac and I can still build and run companion and the simulator without any issues.

I'm wondering if this is still needed - when building companion you get this warning:
   'Libfox not found, simu target will not be available'
but everything still seems to work ok.

**Comment by raphaelcoeffic** [comment, marked as answer]
Libfox is only required for the very simple `simu` target. This mainly used by devs (at least @3djc, @gagarinlg and myself).

**Comment by gagarinlg** [comment]
Yes, but I would like to get rid of it and use SDL, to be able to compile simu with emscripten.
The UI inputs from libfox are used for simu, so when we create some replacements in SDL and redo the key bindings, we should be fine.
Due to the amount of things that need to be done, this is pretty low on my prio list, currently.
